apps/jobs/models.py

```diff
@@ -148,3 +148,12 @@ class Meta:
     is_deleted = models.BooleanField(default=False, null=True)
     is_active = models.BooleanField(default=True, null=True)
     employer_id = models.UUIDField(null=False, editable=False, default=None)
+
+
+class ContactMessage(models.Model):
```

Comment:
specify your database table name for this model here. 
For your reference, you can check out other models `Meta` class.

Reply:
Table name specified in the meta class